lib/rack/session/cookie.rb

```diff
@@ -48,11 +48,17 @@ module Session
     class Cookie < Abstract::Persisted
       # Encode session cookies as Base64
       class Base64
+        def self.strict_encode(str)
+          [str].pack('m0')
+        end
+
         def encode(str)
-          [str].pack('m')
+          Base64.strict_encode(str)
         end
 
         def decode(str)
+          str.unpack('m0').first
+        rescue ArgumentError
```

Comment:
Ah, I see what you mean. Good call as this method can be totally left alone then!

I can amend my commit to not touch this method. Thanks for the feedback!